### PR TITLE
Add additional topics to blockstream listener

### PR DESCRIPTION
**Pull request by justinbarry**
Also ran tslint fix.

@nuevoalex There is still a type issue here. We are storing additional topics as an array of arrays. Can they be flattened or do they need to be maintained?

### CODE REVIEW

**Review by nuevoalex — APPROVED** [review]
Looks good. Are there any tests for blockstream yet?

**Comment by justinbarry** [comment]
After speaking with @MicahZoltu it became clear we need to "de-duplicate" the filter sent to blockstream to minimize eth node requests and unpack/filter the payload on the callback side.

Also, this PR shows proves I don't know how additional topics work. The nested arrays in the filter serve as an OR mechanism. I'm flattening them all here.

File: packages/augur-sdk/src/state/db/BlockAndLogStreamerListener.ts [hunk 1]
```diff
@@ -1,77 +1,122 @@
...
+  ) => string;
 }
 
 export interface BlockAndLogStreamerListenerDependencies {
   address: string;
   blockAndLogStreamer: BlockAndLogStreamerInterface<Block, ExtendedLog>;
   getBlockByHash: (hashOrTag: string) => Promise<Block>;
   eventLogDBRouter: EventLogDBRouter;
-  getEventTopics: (eventName: string) => Array<string>;
+  getEventTopics: (eventName: string) => string[];
```
Comment (MicahZoltu) [inline packages/augur-sdk/src/state/db/BlockAndLogStreamerListener.ts:38]: I read at some point somewhere that `Array<string>` is the preferred/recommended syntax here, `string[]` is supported for legacy/backward compatibility reasons or something.

File: packages/augur-sdk/src/state/db/BlockAndLogStreamerListener.ts [hunk 2]
```diff
@@ -1,77 +1,122 @@
...
     const dependencies = new EthersProviderBlockStreamAdapter(provider);
-    const blockAndLogStreamer = new BlockAndLogStreamer<Block, ExtendedLog>(dependencies.getBlockByHash, dependencies.getLogs, (error: Error) => {
-      console.error(error);
-    });
+    const blockAndLogStreamer = new BlockAndLogStreamer<Block, ExtendedLog>(
+      dependencies.getBlockByHash,
+      dependencies.getLogs,
+      (error: Error) => {
+        console.error(error);
+      }
```
Comment (MicahZoltu) [inline packages/augur-sdk/src/state/db/BlockAndLogStreamerListener.ts:78]: You can just have the parameter here be `console.error`, no need to wrap it in a lambda.

**Comment by MicahZoltu** [comment]
Recommend splitting this into multiple PRs, one that does the formatting changes and another that does the actual code changes.  With the two mixed, it is difficult to identify where the changes are.

**Comment by justinbarry** [comment]
Already done in another PR. We cannot really merge multiple filters in an advantageous way.

**Pull request closed without merging**